spaghetti-core/src/main/java/com/prezi/spaghetti/ast/EnumNode.java

```diff
@@ -2,4 +2,5 @@
 
 public interface EnumNode extends AnnotatedNode, QualifiedTypeNode, ReferableTypeNode {
 	NamedNodeSet<EnumValueNode> getValues();
+	NamedNodeSet<EnumValueNode> getNormalizedValues();
```

Comment:
How about we never return the non-normalized stuff? AST consumers should never be able to access the non-normalized values.